mu-plugins/blocks/multisite-latest-posts/multisite-latest-posts.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Plugin Name: Multisite Latest Posts
+ * Description: A block for use across the whole wp.org network.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\Multisite_Latest_Posts;
+
+add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\register_assets', 20 );
+add_action( 'init', __NAMESPACE__ . '\multisite_latest_posts_block_init' );
+
+/**
+ * Renders the `wporg/mulsite-latest-posts` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the event year for the current post.
+ */
+function render_block( $attributes ) {
+	return sprintf(
+		'<div 
+			class="wporg-multisite-latest-posts-js"
+			data-endpoint="%1$s" 
+			data-items-to-show="%2$s" 
+		></div>',
+		esc_attr( $attributes['endpoint'] ),		
+		esc_attr( $attributes['itemsToShow'] )
+	);
+}
+
+
+/**
+ * Register scripts, styles, and block.
+ */
+function register_assets() {
+	$deps_path = __DIR__ . '/build/index.asset.php';
+	
+	if ( ! file_exists( $deps_path ) ) {
+		return;
+	}
+
+	$block_info = require $deps_path;
+
+	if ( ! is_admin() ) {
+		wp_enqueue_script(
+			'wporg-multisite-latest-post',
+			plugin_dir_url( __FILE__ ) . 'build/front.js',
+			$block_info['dependencies'],
+			$block_info['version'],
+			true
+		);
```

Comment:
Technically this has already been registered by the `block.json` registration, but [since we have a `render_callback` ](https://github.com/WordPress/wordpress-develop/blob/e6f752026650b26152c3b420643c6caade624033/src/wp-includes/class-wp-block.php#L266)it's not enqueued.

If you update the render callback to also accept the rest of the args, you can enqueue the registered script like this:

```php
function render_block( $attributes, $content, $block ) {
	wp_enqueue_script( $block->block_type->view_script );
```

It would be better to do that anyway, since there are more deps for `index` that would get loaded on the frontend for no reason.